src/components/common/SignUp.js

```diff
@@ -46,6 +46,10 @@ const tailFormItemLayout = {
   },
 };
 
+// Test Login User:
+// llama001@maildrop.cc
+// Test001Test
+
```

Comment:
should remove these from the source code

Reply:
Done